### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ripecosta**
## What does this change?
Updates the README to offer some context of what this repo is about. It will be expanded further once the system is a little bit more fleshed out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
 # payment-failure-comms
-Emits payment failure related Custom Events to Braze from Salesforce
+This repo contains a a lambda that emits payment failure related Custom Events to Braze. It fetches data from Salesforce to determine users entering or exiting payment failure and places that information into Braze using Custom Events.
```
Comment (paulbrown1982) [inline README.md:2]: Typo: double a

Comment (ripecosta) [reply README.md]: Fixed!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+
+Its aim is to decouple payment failure comms from payment retries by providing Braze with the information needed to trigger these comms.
```
[the remark this answers is not in the document]
Comment (ripecosta) [reply README.md]: Suggestion added.

**Review by paulbrown1982 — APPROVED** [review]
👍

**Review by johnduffell — COMMENTED** [review]
nice work, useful to have the links and context 👍

**Pull request merged**